cmd/tempo/app/app.go

```diff
@@ -78,6 +80,7 @@ func (c *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet) {
 	c.Distributor.RegisterFlagsAndApplyDefaults(tempo_util.PrefixConfig(prefix, "distributor"), f)
 	c.Ingester.RegisterFlagsAndApplyDefaults(tempo_util.PrefixConfig(prefix, "ingester"), f)
 	c.Querier.RegisterFlagsAndApplyDefaults(tempo_util.PrefixConfig(prefix, "querier"), f)
+	c.Frontend.ApplyDefaults()
```

Comment:
are there any cli flags we care about here?  or are we content with everything being in the config file only?

Reply:
I think we're good on the cli flags, nothing that needs configuration for most use cases. Eventually we might want one for `max_outstanding_per_tenant` but the default (100) works for now